sections/pages/fashion-and-styles/index.js

```diff
@@ -1,5 +1,18 @@
 import SearchFilterSection from "./SearchFilterSection";
 import HeroSection from "./HeroSection";
-import BlogListSection from "./BlogListSection";
+import BlogLists from "./BlogLists";
+import SingleBlogTitleSection from "./single-blog/SingleBlogTitleSection";
+import SingleBlogHeroSection from "./single-blog/SingleBlogHeroSection";
+import SingleBlogContentSection from "./single-blog/SingleBlogContentSection";
+import SingleBlogArticle from "./SingleBlogArticle";
+SingleBlogArticle;
 
-export { SearchFilterSection, HeroSection, BlogListSection };
+export {
+  SearchFilterSection,
+  HeroSection,
+  BlogLists,
+  SingleBlogTitleSection,
+  SingleBlogHeroSection,
+  SingleBlogContentSection,
+  SingleBlogArticle,
```

Comment:
instead, if you feel it's better you can do it like this "export { default as BlogLists  } from './BlogLists'"
both importing it and exporting in one line, instead of manually importing then exporting

Reply:
Alright, I think your suggestion is a better approach, thanks.